queue/abstract/driver/fifo.lua

```diff
@@ -23,25 +23,30 @@ function tube.create_space(space_name, opts)
     local space_opts         = {}
     local if_not_exists      = opts.if_not_exists or false
     space_opts.temporary     = opts.temporary or false
-    space_opts.if_not_exists = if_not_exists
     space_opts.engine        = opts.engine or 'memtx'
     space_opts.format = {
         {name = 'task_id', type = num_type()},
         {name = 'status', type = str_type()},
         {name = 'data', type = '*'}
     }
 
-    local space = box.schema.create_space(space_name, space_opts)
-    space:create_index('task_id', {
-        type = 'tree',
-        parts = {1, num_type()},
-        if_not_exists = if_not_exists
-    })
-    space:create_index('status',  {
-        type = 'tree',
-        parts = {2, str_type(), 1, num_type()},
-        if_not_exists = if_not_exists
-    })
+    local space = box.space[space_name]
+    if if_not_exists and space then
+        -- Validate the existing space.
+        validate_space(box.space[space_name])
```

Comment:
We can reduce indentation level (and diff too) if we'll add `return space` here. Up to you.

Reply:
Yep. Applied.